src/sentry/models/event.py

```diff
@@ -638,6 +639,12 @@ def next_event_id(self, environments=None):
         if 'error' in result or len(result['data']) == 0:
             return None
 
+        # Increment count if the next event ID is after utcnow
+        event_time = parse_date(result['data'][0]['timestamp']).replace(tzinfo=pytz.utc)
+
+        if event_time > datetime.utcnow().replace(tzinfo=pytz.utc):
+            metrics.incr('events.future_next_event_id', skip_internal=False)
```

Comment:
is there a specific reason for skip_internal=False ?

Reply:
No, I wasn't quite sure about this one so just copied what most of the others seemed to do